Diplomacy Screen right table is scrollable

r3versi opened the pull request.

yairm210 commented:
This is not the right way of doing this :)
This leads to the trade options not being mouse scrollable, because they're scrolls-within-scrolls

Add a scrollpane on the specific take you're adding to the right side, instead (like what exists for the major civs - I assume you want this for the city States as well, now)

r3versi commented:
> This is not the right way of doing this :)
> This leads to the trade options not being mouse scrollable, because they're scrolls-within-scrolls
> 
> Add a scrollpane on the specific take you're adding to the right side, instead (like what exists for the major civs - I assume you want this for the city States as well, now)

I think I fixed it the proper way now.  I also added some wrapping to the text so that it is pleasant also with 750x500 resolution. 
 
![Screenshot from 2020-10-14 21-31-18](https://user-images.githubusercontent.com/17114100/96036345-9dafe180-0e64-11eb-9603-6678ee60241f.png)

yairm210 commented on core/src/com/unciv/ui/trade/DiplomacyScreen.kt:
```diff
@@ -137,8 +137,8 @@ class DiplomacyScreen(val viewingCiv:CivilizationInfo):CameraStageBaseScreen() {
-        diplomacyTable.add(friendBonusLabel).row()
-
+                .apply { setWrap(true); setAlignment(Align.center) }
+        diplomacyTable.add(friendBonusLabel).width(650f).row()
```
Please use stage.width-dependant widths, so it will look good both on Android and on higher-resolution PC 🔢

r3versi replied:
got it. i set the width to the one of `rightSideTable` minus some space of padding
